v7/import_blogger/import_blogger.py

```diff
@@ -154,8 +156,11 @@ def import_item(self, item, out_folder=None):
 
         if link_path.lower().endswith('.html'):
             link_path = link_path[:-5]
+        link_path = link_path.lstrip('/')
 
-        slug = utils.slugify(link_path)
+        out_path = os.path.join(self.output_folder, out_folder, link_path)
+        link_fragments = link_path.split('/')
+        slug = utils.slugify(link_fragments[-1])
```

Comment:
Right.  I was thinking Nikola needed to have unique slugs, but that's not the case. This should be fine.